shared/chat/inbox/container.js

```diff
@@ -41,16 +43,32 @@ const getRows = createImmutableEqualSelector([filteredInbox, getPending], (inbox
   return I.List(pending.keys()).concat(inbox)
 })
 
-export default connect(
-  (state: TypedState) => ({
-    isLoading: state.chat.get('inboxUntrustedState') === 'loading',
-    showNewConversation: state.chat.inSearch && state.chat.inboxSearch.isEmpty(),
-    rows: getRows(state),
-  }),
-  (dispatch: Dispatch) => ({
-    loadInbox: () => dispatch(loadInbox()),
-    onNewChat: () => dispatch(newChat([])),
-    onUntrustedInboxVisible: (converationIDKey, rowsVisible) =>
-      dispatch(untrustedInboxVisible(converationIDKey, rowsVisible)),
+// Inbox is being loaded a ton by the navigator for some reason. we need a module-level helper
+// to not call loadInbox multiple times
+const throttleHelper = throttle(cb => cb(), 60 * 1000)
```

Comment:
we can't just make this in mapDispatchToProps cause thats remade when this whole thing gets mounted again :(